### ISSUE TITLE
[Docs]: [Bug]: My page is failing an accessibility audit after installing the cookie consent.  The message seems to indicate that the cookie consent is using an iframe. 

### ISSUE THREAD

**Issue by brblair** [orestbida/cookieconsent#595]
### What is the improvement/update you wish to see?

I might be helpful to mention this is the documentation.  I did see the reference to iframe blocking (as well as the other code for doing so) and perhaps this would help resolve this.  I need to dig in a little more to understand if that is the case.  @orestbida I've been really impressed with your project.

![image](https://github.com/orestbida/cookieconsent/assets/51764103/29ec2736-860b-4a9b-86dd-2e1eb8bf8c1e)

**Comment by orestbida** [comment]
@brblair there is no iframe generated by this plugin, it must be something else on your website.

**Comment by brblair** [comment]
Thanks for confirming @orestbida.  I will investigate further.
